doc/source/whatsnew/v1.1.2.rst

```diff
@@ -17,6 +17,7 @@ Fixed regressions
 - Regression in :meth:`DatetimeIndex.intersection` incorrectly raising ``AssertionError`` when intersecting against a list (:issue:`35876`)
 - Fix regression in updating a column inplace (e.g. using ``df['col'].fillna(.., inplace=True)``) (:issue:`35731`)
 - Performance regression for :meth:`RangeIndex.format` (:issue:`35712`)
+- Fix regression in invalid cache after an indexing operation (:issue:`35521`)
```

Comment:
can you elaborate a bit here, this is pretty general

Reply:
If you have a suggestion, happy to change it. But it's a rather obscure issue that is hard to explain in understandable terms

Reply:
OK, that's nice, thanks!